common/traits/000_wc_leadership_traits.txt

```diff
@@ -10,22 +10,26 @@ archmagus_leader = {
 	customizer = no
 
 	command_modifier = {
+		speed = 0.05	#Time magic
 		actual_light_infantry = 0.2
+		archmages = 0.2
```

Comment:
Use ONLY base unit modifiers (pikemen, light_cavalry, knights, actual_light_infantry, archers and actual_heavy_infantry)
<unit> modifier DON'T WORK WITH SPECIAL UNITS